stable/spark/values.yaml

```diff
@@ -8,31 +8,37 @@ Spark:
 
 Master:
   Name: master
-  Image: "k8s.gcr.io/spark"
-  ImageTag: "1.5.1_v3"
+  Image: "dbanda/spark2.4"
+  ImageTag: "v3"
```

Comment:
What version of spark does this tag represent?

Reply:
This is spark 2.4.0

Reply:
Makes sense. See my https://github.com/helm/charts/pull/11285#pullrequestreview-231267707 below about updating `appVersion`.